src/packet.ml

```diff
@@ -48,6 +48,8 @@ let packet_id_len = 4
 
 let hmac_len = 20 (* SHA1 is what you say *)
 
+let block_size = 16
```

Comment:
Block size of what?

Reply:
renamed to `cipher_block_size` in 47c8b8f